src/test/resources/Features/CMSIDsLookup.feature

```diff
@@ -0,0 +1,19 @@
+Feature: This feature is to test newly generated/unverifiable CMS IDs are inserted into database and available to look up using CMS ID Reverse Lookup
+
+ Scenario Outline: As an end-user, I want to be able to lookup my listings with the old CMS IDs
+	Given I am an end-user on CMS ID Reverse Lookup page
+	When I look up CMS ID "<CMS_ID>" in the reverse look-up tool
+	Then I should see the listings "<CHPL_ID>" that make up the CMS ID 
+	Examples:
+      |CMS_ID			|CHPL_ID|
+      |0015H8GK6K0ZZB2|14.07.07.2452.VEI1.01.01.0.161111|
+      |0015H8GK6K0ZZB2|15.04.04.2891.Sunr.07.01.1.171201|
+      			
+Scenario Outline: If a user loads their cms widget with any of these sets of listings, and generates a CMS ID then it should return the newer CMS ID
+	Given I am an end user on CHPL home page
+	When I Generate CMS ID using CMS ID widget for same listings as in result of reverse lookup
+	Then the generated CMS ID should be "<CMS_ID>"
+	Examples:
+      |CMS_ID|	
+      |0015H8GK6K0ZZB2|
+            
```

Comment:
This should be something where you pass in a comma separated list of CHPL IDs and the CMS ID that should be generated from those IDs, something like:
`|CHPL-0000,CHPL-3232|14ds0sfdfd|`
and then the step that does the "add to the cms widget" should loop through the CHPL IDs

Reply:
Comma separated list sends as-is value in sendkeys (CHPL-0000,CHPL-3232 as input in search box in case of above example), I took a different approach.